templates/etc/haproxy/peers.cfg.j2

```diff
@@ -0,0 +1,8 @@
+{% if haproxy_peers is defined %}
+{% for haproxy_peer in haproxy_peers %}
```

Comment:
Can you change this to `{% for haproxy_peer in haproxy_peers | default([]) %}`, this saves an `if`